msteams-platform/sbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml

```diff
@@ -0,0 +1,280 @@
+
+        The Teams tab app is created in few seconds.
+
+      In TeamsFX CLI,
+
+      * If you prefer interactive mode, execute `teamsfx` new command, then use the keyboard to go through the same flow as in Visual Studio Code.
+
+      * If you prefer non-interactive mode, enter all required parameters in one command:
+
+        `teamsfx new--interactive false--capabilities "dashboard-tab"--programming-language "TypeScript"--folder "./"--app-name dashboard-cli-001`
+
+        After you successfully created the Teams app, start local debugging via F5 in Visual Studio Code. 
+      
+        Select **Debug (Microsoft Edge)** or **Debug (Chrome)** option of your preferred browser, you'll see a tab app loaded:
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/dashboard.png" alt-text="Screenshot shows user profile information and recent Teams activity.":::
+
+        Teams app also supports different themes, including dark and high contrast themes.
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/dark-light-theme.png" alt-text="Screenshot shows the contrast themes.":::
+       
+   
+- title: Add a widget to the dashboard
+  durationInMinutes: 5
+  content: |
+
+     Following are the steps to add a new widget to the dashboard:
+
+     1. [Define a data model](#define-a-data-model)
+     2. [Create a data retrieve service](#create-a-data-retrieve-service)
+     3. [Create a widget file](#create-a-widget-file)
+     4. [Add the widget to the dashboard](#add-the-widget-to-the-dashboard)
+    
+     ## Define a data model
+
+     1. In Visual Studio Code, under **EXPLORER**, go to tabs > src > models.
+     1. Create a **sampleModel.ts** file.
+     1. Use the following code and add it to the **sampleModel.ts** file:
+
+        ```
+        export interface SampleModel {
+         content: string;
+         }
+
+        ```
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/create-sample-model.png" alt-text="Screenshot shows the creation of sampleModel file under the models folder in Visual Studio Code. ":::
+
+
+     ## Create a data retrieve service
+
+     1. In Visual Studio Code, under **EXPLORER**, go to **tabs > src > data**.
+     1. Create a **sampleData.json** file.
+     1. Use the following code in the **sampleData.json** file:
+
+        ```
+        {
+          "content": "Hello world!"
+        }
+        ```
+
+        :::image type="content" source="~/assets/images/sbs-create-a-new-dashboard/create-sample-data-json.png" alt-text="Screenshot shows the creation of sampleData.json file under the data folder in Visual Studio Code.":::
+
+     **To retrieve services**:
+
+     1. In Visual Studio Code, under **EXPLORER**, go to EXPLORER> tabs > src > data> services.
+     1. Create a **sampleService.ts** file.
+     1. Use the following code in the **sampleService.ts**:
+
+        ```
```

Comment:
I think it would be better to specify the language of the following code, so that the code supports syntax highlighting.

Reply:
Updated.

Reply:
Rectified.